### ISSUE TITLE
update MIRI top donors

### ISSUE THREAD

**Issue by riceissa** [vipulnaik/donations#20]
see https://intelligence.org/topcontributors/

**Comment by riceissa** [comment]
Several top donors are new and there might be updates to the values of existing top donors. See for instance this new post: https://intelligence.org/2017/12/14/end-of-the-year-matching/

I think it makes sense to wait until the fundraiser concludes to to update the donors here.

**Comment by vipulnaik** [comment]
Some individual donations that we might want to include, and then subtract off those individual donations from the total amount reported for that individual. The individual donations give better time granularity and therefore make other derived metrics more accurate.

Zvi Mowshowitz: https://thezvi.wordpress.com/2017/12/17/i-vouch-for-miri/ $10,000 compared to $15,010 total listed for money donated since 2015
Jacob Falkovich: https://putanumonit.com/2017/12/10/worried-about-ai/ doesn't seem to give a concrete number (but read more carefully), but $5,415 listed for total money donated since 2015

Also the following individual donations mentioned at https://intelligence.org/2017/12/14/end-of-the-year-matching/ : Christian Calderon ($368K donated thru BTC), Marius van Voorden ($59K donated thru BTC).

**Comment by riceissa** [comment]
Work in progress https://github.com/riceissa/miri-top-contributors

**Comment by riceissa** [comment]
Comparison of donations recorded in DLW (db) vs donations that can be inferred from looking at Internet Archive snapshots (web): https://gist.github.com/riceissa/2f213d7a58aa5edd6488ab69c4fd871d

**Comment by riceissa** [comment]
Here is every donation that can be inferred from snapshots of the top donors page https://gist.github.com/riceissa/0d04ce0584bd5fbe70fbe768faa23d18

There are a lot of donations that appear and disappear (you can see these as donations of negative amounts in the gist).

I can think of several reasons for this but can't tell which reason it is:

- The donation is made in cryptocurrency or something else that's not cash, and as MIRI sells or keeps part of it the value fluctuates a bit
- The donation happens, then the donor requests that their donation not appear on the top contributors page, so it is removed
- For people affiliated with MIRI, if the donation happens before they are employed, their donation can be considered to "cancel out" some of their salary. This doesn't happen in all cases though (e.g. Nate Soares is still listed as a top donor).

**Comment by riceissa** [comment]
There was an issue with Brian Tomasik's donations being duplicated, but I fixed that in 8c2baab359ae215ccff144dc4acbc14a8266fc19.

Also, for the donation date precision, I used 'year' for most of them (and 'multi-year' for the first snapshot, since we only know it happened before the beginning of 2015). But in cases where the snapshots are close together, we can probably use 'quarter' or something. And if we run the script each month, we can say 'month'.
